src/bin.js

```diff
@@ -19,6 +15,7 @@ require('yargs')
   })
   .option('all', {
     type: 'boolean',
+    default: undefined, // necessary because "conflicts" is stupid with booleans
```

Comment:
I don't understand this comment

Reply:
in yargs, a required boolean argument defaults to false, so there’s no way to determine if the user actually passed it or not. Defaulting it to `undefined` hacks around this behavior.